dhis2/capture-app
# feat: DHIS2-9808 Capture TEType id in the url

https://jira.dhis2.org/browse/DHIS2-9808

Here I am adding the `trackedEntityTypeId` in the url and also in the redux store. Each time a user changes their selection from the `TrackedEntityTypeSelector` this will be reflected in the url.


## How to test

Try to use the `TrackedEntityTypeSelector` from either the Search or New page and see if the url is changing.
